src/debug.c

```diff
@@ -1960,6 +1960,9 @@ void logStackTrace(void *eip, int uplevel) {
     /* Write symbols to log file */
     ++uplevel;
     writeStacktraces(fd, uplevel);
+    msg = "\n===== Done logging stacktraces =====\n";
```

Comment:
the header is:
```
    msg = "\n------ STACK TRACE ------\n";
```
so it makes sense that the footer will have a similar format.
i.e. `-` instead of `=`, upper case.
maybe
```suggestion
    msg = "\n------ STACK TRACE DONE ------\n";
```